specs/core/0_beacon-chain.md

```diff
@@ -1460,8 +1432,8 @@ def process_attester_slashing(state: BeaconState, attester_slashing: AttesterSla
     assert is_valid_indexed_attestation(state, attestation_2)
 
     slashed_any = False
-    attesting_indices_1 = attestation_1.custody_bit_0_indices + attestation_1.custody_bit_1_indices
-    attesting_indices_2 = attestation_2.custody_bit_0_indices + attestation_2.custody_bit_1_indices
+    attesting_indices_1 = attestation_1.attesting_indices
+    attesting_indices_2 = attestation_2.attesting_indices
```

Comment:
Functionally all changes in the phase 0 doc look good. Have some doubt about style here, but avoiding longer lines is good too.

Reply:
yeah, I'll clean this up. was a relic of the past